bin/run-browser-tests

```diff
@@ -13,6 +13,7 @@ docker volume create browser-tests-node-modules
 # Additionally we map node_modules to a separate volume so that it doesn't
 # conflict with node_modules created locally.
 docker run --rm -it \
+  --platform linux/x86_64 \
```

Comment:
Can you clarify why is this needed? I thought platform is the part of the image itself? Though I don't understand docker that well.

Reply:
This was an attempt to get the browser tests to run on my M1 arm64 Mac. It does allow the docker image to startup, but the test fails later anyway. I'll remove this.